src/axis.rs

```diff
@@ -1,14 +1,17 @@
-use std::rc::Rc;
-
-// Axis scaled value, expected to be between 0 and 1
-// except in the case where the value is outside of the axis range
+/// Axis scaled value, expected to be between 0 and 1
+/// except in the case where the value is outside of the axis range
 pub struct NormalisedValue(pub f32);
 
+/// A trait for creating an axis scale. It has two functions:
+/// `ticks` returns a vector of axis ticks
+/// `normalise` normalises the input `value`
 pub trait AxisScale {
-    fn ticks(&self) -> Rc<Vec<AxisTick>>;
+    fn ticks(&self) -> Vec<AxisTick>;
     fn normalise(&self, value: f32) -> NormalisedValue;
 }
 
+/// An axis tick
```

Comment:
Full stop?

Reply:
Expanded on the documentation including justifications